network-api/networkapi/wagtailpages/templates/wagtailpages/research_library_page.html

```diff
@@ -52,11 +52,10 @@ <h1>{{ page.title }}</h1>
         </div>
         <div class="tw-h4-heading tw-mb-0">
           {# RESULTS COUNT #}
-          <strong>{{ research_detail_pages_count }}</strong>
           {% blocktranslate count counter=research_detail_pages_count trimmed %}
-            result
+            <strong>{{ research_detail_pages_count }}</strong> result
           {% plural %}
-            results
+            <strong>{{ research_detail_pages_count }}</strong> results
           {% endblocktranslate %}
```

Comment:
**Question** Why do we need this duplication?

Reply:
Can this also include other Django template tags? Because in that case we might need to rest of that sentence too. It's "{{ number of results }} results for {{ search query }}".